Software/Python/ir_remote_control/gobox_ir_receiver_libs/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#Install the ir_receiver library systemwide
+sudo python /home/pi/Desktop/GoPiGo/Software/Python/ir_remote_control/gobox_ir_receiver_libs/setup.py install
```

Comment:
any suggestions for a better way. I also realized that the paths were pretty long when I started writing the script